.github/workflows/slovensko_digital_ci.yml

```
@@ -0,0 +1,21 @@
+name: Slovensko.Digital CI
+
+on: 
+  push:
+    branches:
+      - master
+
+jobs:
+  deploy:
+    if: ${{github.ref == 'refs/heads/master'}}
+
+    runs-on: ubuntu-latest
```

tato kontrola je uz vyssie (`branches: master`)

```suggestion
    runs-on: ubuntu-latest
```